cenc-format.html

```diff
@@ -382,157 +374,92 @@ <h2 id="w3c-editor-s-draft-10-december-2015"><abbr title="World Wide Web Consort
   <hr title="Separator for header">
 </div>
     <!-- TODO: Use styles for ISO boxes like the MSE registry. -->
-    <section id="abstract" class="introductory" property="dc:abstract"><h2 id="h-abstract" resource="#h-abstract"><span property="xhv:role" resource="xhv:heading">Abstract</span></h2>
-      <p>This specification defines the stream format and initialization data for using ISO Base media File Format [<cite><a class="bibref" href="#bib-ISOBMFF">ISOBMFF</a></cite>] content using the ISO Common Encryption ('cenc') protection scheme [<cite><a class="bibref" href="#bib-CENC">CENC</a></cite>] with the <a href="index.html#">Encrypted Media Extensions</a>.
+    <section property="dc:abstract" class="introductory" id="abstract"><h2 resource="#h-abstract" id="h-abstract"><span property="xhv:role" resource="xhv:heading">Abstract</span></h2>
+      <p>This specification defines the stream format and initialization data for using ISO Base media File Format [<cite><a href="#bib-ISOBMFF" class="bibref">ISOBMFF</a></cite>] content using the ISO Common Encryption ('cenc') protection scheme [<cite><a href="#bib-CENC" class="bibref">CENC</a></cite>] with Encrypted Media Extensions [<cite><a href="#bib-encrypted-media" class="bibref">encrypted-media</a></cite>].
       It also defines a common SystemID and PSSH box format for use with Encrypted Media Extensions.
       </p>
-    </section><section id="sotd" class="introductory"><h2 id="h-sotd" resource="#h-sotd"><span property="xhv:role" resource="xhv:heading">Status of This Document</span></h2>
+    </section><section id="sotd" class="introductory"><h2 resource="#h-sotd" id="h-sotd"><span property="xhv:role" resource="xhv:heading">Status of This Document</span></h2>
   
     
       
         <p>
-          <em>This section describes the status of this document at the time of its publication. Other documents may supersede this document. A list of current <abbr title="World Wide Web Consortium">W3C</abbr> publications and the latest revision of this technical report can be found in the <a href="http://www.w3.org/TR/"><abbr title="World Wide Web Consortium">W3C</abbr> technical reports index</a> at http://www.w3.org/TR/.</em>
+          This document is merely a <abbr title="World Wide Web Consortium">W3C</abbr>-internal  document. It
```

Comment:
The use of specStatus: "base" in ReSpec causes the SotD to say this is an internal document with no standing. At least the former seems incorrect.